### PR TITLE
feature/#1574 implement CSS in JS using Emotion

### PR DESCRIPTION
**Pull request by mcespo**
This update removes SASS in favor of Emotion as a means to style the site. There are a lot of changes here so testing and verifying all is still visually intact is exceptionally important. Overall site functionality and behavior should not be impacted by this change. 

Will make small incremental updates to this work (including font loading, color refinement, scaling fonts, and other goodies in following PRs).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,6 @@
 .*/*
```
Comment (mcespo) [inline .gitignore:1]: no need to keep sass directories in our ignore

File: _config.yml [hunk 2]
```diff
@@ -12,40 +12,42 @@ collections:
...
 autoOpenMessage: This page automatically opens a drafted email for you.
-badMailtoMessage: Email link not working correctly? You can copy and paste the recipients,
+badMailtoMessage:
+  Email link not working correctly? You can copy and paste the recipients,
   subject and message individually below.
 url: https://defund12.org
 logoUrl: "/images/defund12.org.png"
 faviconUrl: "/favicon.svg"
 metaPreviewUrl: https://defund12-image-api.now.sh/api/preview/
-sass:
```
Comment (mcespo) [inline _config.yml:46]: removing sass refernces

File: _config.yml [hunk 3]
```diff
@@ -12,40 +12,42 @@ collections:
     title: Uploads
     output: true
 defaults:
```
Comment (mcespo) [inline _config.yml:14]: this is a format consistency change

File: package.json [hunk 4]
```diff
@@ -1,79 +1,81 @@
 {
```
Comment (mcespo) [inline package.json:1]: format change

File: src/components/common/Layout.tsx [hunk 5]
```diff
@@ -115,3 +117,162 @@ export default function Layout(
     />
   );
 }
```
Comment (mcespo) [inline src/components/common/Layout.tsx:119]: the reset and overall site layout will now live in the layout file

File: src/components/email/Email.tsx [hunk 6]
```diff
@@ -226,3 +226,51 @@ export const pageQuery = graphql`
     }
   }
 `;
+
+const StyledSection = styled.section`
```
Comment (mcespo) [inline src/components/email/Email.tsx:230]: our site doesn't really need media queries anymore. Outside of a few edge-cases, we can just use basic calculations to output our layout

File: src/components/letter/MyAddressInput.tsx [hunk 7]
```diff
@@ -102,9 +102,21 @@ export default function MyAddressInput({
...
               />
             </div>
-          </div>
+          </Div>
         </div>
       </fieldset>
     </>
   );
 }
+
```
Comment (mcespo) [inline src/components/letter/MyAddressInput.tsx:111]: this is an override for some of the field styles here. I will come back around to fix this. I appreciate the work that was done and am grateful that we were able to use a class system that was quick to implement. That said, I will always be your CSS friend who can solve the needs of most cases without the need for libraries (so long as my schedule allows of course)

File: src/components/template-list/TemplateList.tsx [hunk 8]
```diff
@@ -109,16 +110,18 @@ class _TemplateList extends React.Component<
               )
               .map((stateGroup) => {
                 return (
-                  <ul key={uuid()} className="state">
+                  <>
```
Comment (mcespo) [inline src/components/template-list/TemplateList.tsx:113]: we should not have h2s inside an unordered (or any) list

**Comment by Ceshion** [comment]
just from looking through the source changes this looks like a good change, but I will leave a full review to others on account of a lack of time today 😊 thank you for your work @mcespo!

**Pull request closed without merging**